anyblok/tests/test_many2one.py

```diff
@@ -305,7 +305,7 @@ class Person:
     Selection,
 ]
 
-if not sgdb_in(['MySQL', 'MariaDB']):
+if not sgdb_in(['MySQL', 'MariaDB', 'MsSQL']):
     params.append(DateTime)
     params.append(URL)
```

Comment:
Are sure that the column URL does not work this MsSQL

Reply:
You are right because the test_column tests are OK for URL.
I think it's a problem concerning the primary key, MsSQL is not okay with NVARCHAR
Here is the traceback :
```
sqlalchemy.exc.OperationalError: (pymssql.OperationalError) (1919, b"Column 'id' in table 'address' is of a type that is invalid for use as a key column in an index.DB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\nDB-Lib error message 20018, severity 16:\nGeneral SQL Server error: Check messages from the SQL Server\n")
E   [SQL: 
E   CREATE TABLE address (
E       id NVARCHAR(max) NOT NULL, 
E       CONSTRAINT anyblok_pk_address PRIMARY KEY (id)
E   )
E   
E   ]
```